pkg/networkmanager/dialogs-common.jsx

```diff
@@ -254,14 +254,12 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
     }
 
     return (
-        <>
-            <Button id={"networking-" + (!iface ? "add-" : "edit-") + type}
+        <Button id={"networking-" + (!iface ? "add-" : "edit-") + type}
                     isInline={!!iface}
                     onClick={syn_click(model, show)}
                     variant={!iface ? "secondary" : "link"}>
```

Comment:
Please fix the indentation.

Reply:
Seems this comes up too much :) I think we should really let eslint handle this better but that requires some starter-kit / podman